compiler/rustc_codegen_llvm/src/asm.rs

```diff
@@ -259,7 +259,9 @@ impl AsmBuilderMethods<'tcx> for Builder<'a, 'll, 'tcx> {
                 InlineAsmArch::RiscV32 | InlineAsmArch::RiscV64 => {}
                 InlineAsmArch::Nvptx64 => {}
                 InlineAsmArch::Hexagon => {}
-                InlineAsmArch::Mips => {}
+                InlineAsmArch::Mips | InlineAsmArch::Mips64 => {
+                    constraints.push("~{FCSR}".to_string());
```

Comment:
No clobber is needed here. Also LLVM doesn't recognize `FCSR` as a valid register name.

Reply:
that's weird. risc-v has llvm support. I'll change it to a FIXME then.

Reply:
There's no need, clang doesn't insert the clobber either even if you clobber the flags.